helios-client/src/main/java/com/spotify/helios/common/descriptors/Job.java

```diff
@@ -190,6 +192,7 @@ public Job(@JsonProperty("id") final JobId id,
              @JsonProperty("metadata") @Nullable final Map<String, String> metadata) {
     this.id = id;
     this.image = image;
+    this.created = new Date();
```

Comment:
maybe use a millisecond timestamp to be consistent with the timestamps reported in the task status history?